OMCompiler/Makefile.omdev.mingw

```diff
@@ -150,11 +168,11 @@ else
 endif
 else
 	$(eval VSCOMNTOOLS=$(VS120COMNTOOLS))
-	$(eval BOOST_PATH_MSVC=$(OMDEVMSYS)/lib/3rdParty/boost-1_59/)
+	$(eval BOOST_PATH_MSVC=$$(OMDEV)/lib/3rdParty/boost-1_59/)
 endif
 else
 	$(eval VSCOMNTOOLS=$(VS140COMNTOOLS))
-	$(eval BOOST_PATH_MSVC=$(OMDEVMSYS)/lib/3rdParty/boost-1_59/)
+	$(eval BOOST_PATH_MSVC=$$(OMDEV)/lib/3rdParty/boost-1_59/)
 	$(eval VCVARS_PARAMS='x86 8.1')
```

Comment:
I think this might be the cause of the latest issue: 

```
CMake Error at C:/OMDev/tools/msys64/ucrt64/lib/cmake/Boost-1.83.0/BoostConfig.cmake:141 (find_package):

  Found package configuration file:
    C:/OMDev/tools/msys64/ucrt64/lib/cmake/boost_filesystem-1.83.0/boost_filesystem-config.cmake

but it set boost_filesystem_FOUND to FALSE so package "boost_filesystem" is

  considered to be NOT FOUND.  Reason given by package:
  No suitable build variant has been found.

  The following variants have been tried and rejected:

  * libboost_filesystem-mt.dll.a (64 bit, need 32)
  * libboost_filesystem-mt.a (64 bit, need 32)
 ```

I am not sure why it is set to x86. It should be `x64` or `amd_64`. Although I have no idea why the error manifested just now. However, if it was indeed intended to be set to x86, at least that might explain why we were shipping some 32 bit libraries with the 64 bit build.